xcube_sh/store.py

```diff
@@ -74,6 +79,36 @@ def get_open_data_params_schema(self, data_id: str = None) -> JsonObjectSchema:
         return self._get_open_data_params_schema(self._describe_data(data_id))
 
     def open_data(self, data_id: str, **open_params) -> xr.Dataset:
+        """
+        Opens the dataset with given *data_id* and *open_params*.
+
+        Possible values for *data_id* can be retrieved from the :meth:SentinelHubDataStore::get_data_ids method.
+        Possible keyword-arguments in *open_params* are:
+
+        * ``variable_names: Sequence[str]`` - optional list of variable names.
+            If not given, all variables are included.
+        * ``variable_units: Union[str, Sequence[str]]`` - units for all or each variable
+        * ``variable_sample_types: Union[str, Sequence[str]]`` - sample types for all or each variable
+        * ``crs: str`` - spatial CRS identifier. must be a valid OGC CRS URI.
+        * ``tile_size: Tuple[int, int]`` - optional tuple of spatial tile sizes in pixels.
+        * ``bbox: Tuple[float, float, float, float]`` - spatial coverage given as (minx, miny, maxx, maxy)
+            in units of the CRS. Required parameter.
+        * ``spatial_res: float`` - spatial resolution in unsits of the CRS^.
+            Required parameter.
+        * ``time_range: Tuple[Optional[str], Optional[str]]`` - tuple (start-time, end-time).
```

Comment:
ISO 8601 can probably be assumed, but might be as well to make it explicit.

Reply:
What do you want me to do?

Reply:
Explicitly state the format of the start-time and end-time strings and the semantics of an omitted value for either or both of them. (Or link/refer to the store conventions document for details on this.)

Reply:
I fully agree!